checks.d/kubernetes.py

```diff
@@ -46,6 +46,10 @@
     RATE: {True: HISTORATE, False: RATE}
 }
 
+POD_NAME_LABEL = "io.kubernetes.pod_name"
+NAMESPACE_LABEL = "io.kubernetes.pod.namespace"
```

Comment:
I think we should create a dict of labels in `kubeutil` and import it when needed. These 2 are also in `checks.d/docker_daemon.py` and we'll probably add more in the future. It would be better to have them all in a single place.